ui/src/common/QualityInfo.js

```diff
@@ -5,7 +5,8 @@ export const QualityInfo = (props) => {
   let { suffix, bitRate } = props.record
   suffix = suffix.toUpperCase()
   let info = suffix
-  if (bitRate !== '') {
+  let lossLessFormats = ['FLAC', 'WAV', 'DSF']
+  if (!lossLessFormats.includes(bitRate)) {
```

Comment:
You need to check if `includes(suffix)`, not `bitRate`

A test file for this component would be awesome for this component, as it has some simple but tricky logic.

Reply:
It was 2 am in the morning so i messed up. Will correct it

Reply:
Which library we use for writing test files?

Reply:
We use Jest + @testing-library. Take a look at `AboutDialog.test.js` for an example.